nilearn/glm/tests/test_first_level.py

```diff
@@ -2062,6 +2062,21 @@ def test_flm_fit_surface_image(_make_surface_glm_data):
     assert isinstance(model.masker_, SurfaceMasker)
 
 
+def test_warn_flm_smooth_surface_image(_make_surface_glm_data):
+    """Test FirstLevelModel with surface image and mask_img set to False."""
+    mini_img, des = _make_surface_glm_data(5)
+    model = FirstLevelModel(mask_img=False, smoothing_fwhm=5)
+    with pytest.warns(
+        UserWarning,
+        match="Parameter smoothing_fwhm is not supported for surface data",
+    ):
+        model.fit(mini_img, design_matrices=des)
+
+    assert isinstance(model.masker_.mask_img_, SurfaceImage)
+    assert model.masker_.mask_img_.shape == (9,)
+    assert isinstance(model.masker_, SurfaceMasker)
```

Comment:
```suggestion
```

tested in the test above, no?

Reply:
I just wanted to make sure that it still correctly instantiates the masker with the warning, but agreed that it's directly redundant. Will remove